### ISSUE TITLE
[Bug] [DependentNode] running success but log can not output in ui

### ISSUE THREAD

**Issue by SbloodyS** [apache/dolphinscheduler#7395]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

![](https://files.catbox.moe/4yk7n5.png)

```
[INFO] 2021-12-14 13:48:46.925 org.apache.dolphinscheduler.service.process.ProcessService:[1060] - start submit task : dependent_test, instance id:2, state: RUNNING_EXECUTION
[INFO] 2021-12-14 13:48:46.937 org.apache.dolphinscheduler.service.process.ProcessService:[1073] - end submit task to db successfully:2 dependent_test state:SUBMITTED_SUCCESS complete, instance id:2 state: RUNNING_EXECUTION  
[INFO] 2021-12-14 13:48:46.965 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[1186] - remove task from stand by list, id: 2 name:dependent_test
[INFO] 2021-12-14 13:48:52.252 org.apache.dolphinscheduler.server.master.runner.EventExecuteService:[126] - handle process instance : 2 , events count:1
[INFO] 2021-12-14 13:48:52.253 org.apache.dolphinscheduler.server.master.runner.EventExecuteService:[129] - already exists handler process size:0
[INFO] 2021-12-14 13:48:52.253 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[289] - process event: State Event :key: null type: TASK_STATE_CHANGE executeStatus: RUNNING_EXECUTION task instance id: 2 process instance id: 2 context: null
[INFO] 2021-12-14 13:48:52.256 org.apache.dolphinscheduler.server.master.runner.task.DependentTaskProcessor:[178] - dependent item complete :|| 3843088697536-0-day-today,SUCCESS
[INFO] 2021-12-14 13:48:52.256 org.apache.dolphinscheduler.server.master.runner.task.DependentTaskProcessor:[200] - dependent task completed, dependent result:SUCCESS
[INFO] 2021-12-14 13:48:52.266 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[366] - work flow 2 task 2 state:SUCCESS 
[INFO] 2021-12-14 13:48:52.271 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[1137] - work flow process instance [id: 2, name:dependent_test-1-20211214134846902], state change from RUNNING_EXECUTION to SUCCESS, cmd type: START_PROCESS
[INFO] 2021-12-14 13:48:52.282 org.apache.dolphinscheduler.server.master.runner.WorkflowExecuteThread:[422] - process:2 state SUCCESS change to SUCCESS
[INFO] 2021-12-14 13:48:52.301 org.apache.dolphinscheduler.server.master.runner.EventExecuteService:[138] - process instance 2 finished.
```

The log can be output normally in logs/dolphinscheduler-master.log.But can not output normally in ui.

### What you expected to happen

running dependent node success.

### How to reproduce

running dependent node

### Anything else

It's a 2.0.1-release bug.

### Version

dev

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by songjianet** [comment]
I will look at it later.

**Comment by SbloodyS** [comment]
it seems like not an ui problem.thanks. @songjianet
this pr has not been merged into 2.0.1-release #7029

**Comment by lenboo** [comment]
Fixed in 7029 , close it.
